content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst

```diff
@@ -1,137 +1,285 @@
-lots for each operation type. To do so, open the *Inventory* app and
-go to :menuselection:`Configuration --> Operation Types`.
+On this :abbr:`RFQ (request for quotation)`, fill out the necessary information by adding a
+:guilabel:`Vendor`, and adding the desired products to the :guilabel:`Product` lines, by clicking
+:guilabel:`Add a product` (under the :guilabel:`Products` tab).
 
-For each type (receipts, internal transfers, deliveries, …), you can
-decide if you allow to create new lot numbers or only use existing ones.
-By default, the creation of new lots is only allowed at product
-reception, and using existing lot numbers isn’t allowed in this
-operation. If you have inter-warehouse transfers and track products by
-lots, it can be useful to allow using existing lot numbers in receipts
-too.
+Choose the desired quantity of the product to order by changing the number in the
+:guilabel:`Quantity` column.
 
-.. image:: lots/lots_08.png
-    :align: center
+Once the :abbr:`RFQ (request for quotation)` has been filled out, click :guilabel:`Confirm Order`.
+When the :abbr:`RFQ (request for quotation)` is confirmed, it becomes a :guilabel:`Purchase Order`,
+and a :guilabel:`Receipt` smart button appears. Click the :guilabel:`Receipt` smart button to be
+taken to the warehouse receipt form.
+
+.. note::
+   Clicking :guilabel:`Validate` before assigning a lot number to the ordered product quantities
+   will result in a :guilabel:`User Error` pop-up. The pop-up requires entry of a lot or serial
+   number for the ordered products. The :abbr:`RFQ (request for quotation)` **cannot** be validated
+   without a lot number being assigned.
+
+.. image:: lots/lots-user-error-popup.png
+   :align: center
+   :alt: Add lot/serial number user error popup.
+
+From here, click the :guilabel:`Additional Options` menu, represented by a `hamburger` icon (four
+horizontal lines, located to the right of the :guilabel:`Unit of Measure` column in the
+:guilabel:`Operations` tab). Clicking that icon reveals a :guilabel:`Detailed Operations` pop-up.
+
+In this pop-up, configure a number of different fields, including the assignation of a lot number,
+under the :guilabel:`Lot/Serial Number Name` column, located at the bottom of the pop-up.
+
+There are two ways to do this: **manually** and **copy/paste**.
+
+- **Manually assign lot numbers**: Click :guilabel:`Add a line` and choose the location the products
+  will be stored in under the :guilabel:`To` column. Then, type a new :guilabel:`Lot Number Name`
+  and set the :guilabel:`Done` quantity.
+
+.. image:: lots/lots-assign-lot-number-popup.png
+   :align: center
+   :alt: Assign lot number detailed operations popup.
+
+.. note::
+   If quantities should be processed in multiple locations and lots, click :guilabel:`Add a line`
+   and type a new :guilabel:`Lot Number Name` for additional quantities. Repeat until the
+   :guilabel:`Quantity Done` matches the :guilabel:`Demand`.
```

Comment:
These _can_ be but don't _have_ to be indented to align with the bullet point since they are in context of it - same with the bullet point below